src/main/java/com/moodybluez/enterprise/controller/EntryController.java

```diff
@@ -6,10 +6,7 @@
 import org.springframework.format.annotation.DateTimeFormat;
 import org.springframework.web.bind.annotation.*;
 
-import java.util.Arrays;
-import java.util.HashMap;
-import java.util.List;
-import java.util.Map;
```

Comment:
Imports Optimization. This is not a big problem here, but if we had big application with a lots of imports, unnecessary imports may slow down application. So it is best practice to only imports that are required.

Reply:
wildcard imports are generally not recommended, for documentation reasons.  I'd keep it as it were.
If you imported java.util.*, and java.sql.*, and then referenced class Date, which one would it choose?  A different class by that name exists in both packages.